app/services/decline_offer_by_default.rb

```diff
@@ -20,12 +20,14 @@ def call
       end
     end
 
-    application_choices.each do |application_choice|
-      NotificationsList.for(application_choice, event: :declined_by_default, include_ratifying_provider: true).each do |provider_user|
-        ProviderMailer.declined_by_default(provider_user, application_choice).deliver_later
+    unless application_form.continuous_applications?
+      application_choices.each do |application_choice|
+        NotificationsList.for(application_choice, event: :declined_by_default, include_ratifying_provider: true).each do |provider_user|
+          ProviderMailer.declined_by_default(provider_user, application_choice).deliver_later
+        end
       end
-    end
 
-    CandidateMailer.declined_by_default(application_form).deliver_later
+      CandidateMailer.declined_by_default(application_form).deliver_later
```

Comment:
The Trello card, only mentioned to remove the provider email, but it made sense to me to remove the provider email too. Just flagging here incase my sense makes no sense 😆

Reply:
You mean remove the candidate email? Can you flag to Maeve/Myles/James? @JR-G

Reply:
Yes I did @tomas-stefano, will edit my comment! 

@MaeveRoseveare @MylesJarvis @JR-G ☝🏼 is the plan to keep the DBD candidate email in?

Reply:
That is the plan for next cycle, as DBD will not exist any more. These should still remain functioning for this cycle though. We had a separate [ticket for the candidate emails](https://trello.com/c/mPBt2KOJ/483-ca-emails-remove-decline-by-default-dbd-emails) - will let Myles and James advise on what's best in terms of doing candidate + provider together!

We plan to replace these with new emails to nudge candidates to accept offers which I'm currently writing.

Reply:
Thanks @MaeveRoseveare! I'll merge this in tomorrow morning. 

Just found the card for removing the candidate DBD emails: https://trello.com/c/mPBt2KOJ/483-ca-emails-remove-decline-by-default-dbd-emails

Two for the price of one 🙌🏼

Reply:
Yup – @MaeveRoseveare  has covered it & happy for both to go in together!